### DISCUSSION TITLE
jwt for admin does not have permissions

### DISCUSSION THREAD

**Discussion by camelman90** [mozilla/hubs#3442] [Hubs Cloud Questions] [answered]
For some reason my jwtDecode token doesnt return proper permissions. Admin doesnt have any other permissions than normal users, even though permissions object postgres_role is "ret_admin" and I can access the admin panel, I cant fly, close room, join invite rooms, etc. unless I am room owner. 

The account id in the jwt is also the same as in admin panel for the account, and "is admin" is checked in admin panel.

Anyone got some pointers?

**Comment by robinkwilson** [comment, marked as answer]
Only room owners can manage room permissions and NOT admins was a design choice. 
For the use case for hubs.mozilla.com: we didn't want the hubs devs/employees to be able to manage other people's rooms they make on hubs.mozilla.com. 

I'd create a HC feature request with an issue to ask for a toggleable feature to make admins all rooms admins. I'd include a use case with the feature request so we're clear on what you need! Create issues here: https://github.com/mozilla/hubs-cloud/issues

**Reply by camelman90** [reply]
Thank you for the answer. It makes alot of sense. When I hear admin I always assumed all permissions. Took me 6 months of intense hubs cloud development to notice, but now we are more people doing room administration and it became apparent. Will make an issue on hubs-cloud to describe why this would be useful. Thank you! 👍 👍
